src/ui/Skeletos/DuplicateSkeleton/DuplicateSkeleton.tsx

```diff
@@ -0,0 +1,14 @@
+import { ReactElement } from "react"
+
+type Props = {
+  howMany: number
+  children: ReactElement
+}
+
+export const DuplicateSkeleton = (props: Props) => {
```

Comment:
```suggestion
export const DuplicateSkeleton = ({ howMany, children }: Props) => {
```

Reply:
Yes, that might be a good idea. I should go through and switch over to destructured at some point